.github/workflows/promote.yml

```diff
@@ -0,0 +1,28 @@
+name: Promote Release
+
+on:
+  # Triggered by the dist server
+  workflow_dispatch:
```

Comment:
Allow manual runs too :)

Also add documentation on the README.md how this will be externally triggered.

Reply:
`workflow_dispatch` allows for manual runs